otwcode/otwarchive
# AO3-4539 Remove spam verification for logged-in users on the Support, Abuse forms

## Issue

https://otwarchive.atlassian.net/browse/AO3-4539

## Purpose

Before running spam verification, checks if the user is logged in, and the email they submit matches the one on their account.

## Testing Instructions

- Create User with any information and submit both types of forms with generic data without editing the pre-filled fields. Both forms should submit successfully.
- With the same user, attempt to submit the same forms, but this time using the email address "akismet-guaranteed-spam@example.com". This should trigger a spam warning on submission.

- Repeat last 2 steps without logging in. Fill in the empty fields with standard, valid-looking data. Both steps should yield the same results as a logged in user.

- Create another User, with "akismet-guaranteed-spam@example.com" as its email address. Repeat the first step, but this time the forms should accept the "spam" email address.

## Credit

*What name do you want us to use to credit your work in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?*

bird

*What pronouns do you prefer (she/her, he/him, zie/hir etc)?*

they
